lib/item.rb

```diff
@@ -1,5 +1,5 @@
 require 'bigdecimal'
-require_relative 'date_formatter' 
+require_relative 'date_formatter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.